@navikt/ds-css/index.css

```diff
@@ -0,0 +1,13 @@
+@import "accordion/index.css";
```

Comment:
Skal vi anbefale teamene å importere hele styling-pakka, eller bare de komponentene de bruker?

Reply:
Jeg tenker vi anbefaler å importere hele stilpakken da det vil gjøre veldig enkelt for bruker. Hvis vi ønsker at bruker skal kunne bare ta i bruk enkelte komponenter så bør kanskje alle filene bli "minimert" og lagt i `/dist`